modules/zip/src/parse-zip/zip64-info-generation.ts

```diff
@@ -53,10 +53,10 @@ type NumberSetter = (header: DataView, offset: number, value: number | bigint) =
 export function setFieldToNumber(
   header: DataView,
   fieldSize: number,
-  fieldOffset: number,
+  fieldOffset: number | bigint,
   value: number | bigint
 ): void {
-  NUMBER_SETTERS[fieldSize](header, fieldOffset, value);
+  NUMBER_SETTERS[fieldSize](header, Number(fieldOffset), value);
```

Comment:
Doesn't this throw away bigint information? Is that OK?

Reply:
The reason of change there is that in many cases we work with offsets in `bigint` and I just thought that code will be cleaner if this function converts the type itself. `NUMBER_SETTERS` accept only number because it works only with DataView, which offset should be no bigger then 2 Gigabytes = 2,147,483,648 Bytes, so `Number` is fully enough for it.